library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -1085,6 +1088,31 @@ public void onResume() {
         if (warmGps && Permissions.check(getContext(), Manifest.permission.ACCESS_FINE_LOCATION)) {
             checkLocationSettingsAndStartLocationServices(true);
         }
+
+        // Explain the consequence of rejecting enabling location
+        if (isRequestingEnableLocation) {
```

Comment:
Why do you need `isRequestingEnableLocation`

Can't `hasAlreadyRequestedEnableLocation` be used?

Reply:
They don't store the same state. They will both have(and should have) different states when `onResume` is called

Reply:
It's not clear what each does then.

Maybe add comments at the top where the variables are declared to delineate this.

Reply:
Great!